pkg/testing/kubernetes/kind/provisioner.go

```diff
@@ -72,46 +76,33 @@ func (p *provisioner) SetLogger(l runner.Logger) {
 }
 
 func (p *provisioner) Supported(batch define.OS) bool {
-
-	supported := batch.Type == define.Kubernetes && batch.Arch == runtime.GOARCH && (batch.Distro == "" || batch.Distro == "kind")
```

Comment:
the Distro logical check was here to allow a k8s test run only for a specified provider; "" means undefined thus it can run under any provider, "kind" runs only run under a Kind-based k8s cluster. Do we want to remove this specification?

Reply:
True. I will add that back, as I could see a tests saying specifically AKS or GKE or something.